apps/worker/src/batch_announcer/batch.announcer.service.ts

```diff
@@ -43,7 +43,7 @@ export class BatchAnnouncementService extends BaseConsumer implements OnModuleDe
     try {
       const publisherJob = await this.ipfsPublisher.announce(job.data);
       // eslint-disable-next-line no-promise-executor-return
-      await this.publishQueue.add(publisherJob.id, publisherJob, { jobId: publisherJob.id, removeOnComplete: 1000 });
+      await this.publishQueue.add(publisherJob.id, publisherJob, { jobId: publisherJob.id, removeOnComplete: 1000, attempts: 3 });
```

Comment:
we need attempts here since 1% of the time that duplicate nonce error might happen